Makefile

```diff
@@ -0,0 +1,58 @@
+# Makefile for Oakville and Milton Humane Society Project
+.PHONY: lint format migration generate-migration start-docker stop-docker clean-db seed-db read-tables read-table help 
+
+# Linting & Formatting
+lint:
+	docker exec -it humane_society_backend /bin/bash -c "yarn lint"
+	docker exec -it humane_society_frontend /bin/bash -c "yarn lint"
+
+format:
+	docker exec -it humane_society_backend /bin/bash -c "yarn fix"
+	docker exec -it humane_society_frontend /bin/bash -c "yarn fix"
+
+# Database Migrations
+migration:
+	docker exec -it humane_society_backend /bin/bash -c "node migrate up"
+
+generate-migration:
+	docker exec -it humane_society_backend /bin/bash -c "node migrate create --name $(name).ts"
```

Comment:
I think for any of these commands that take in arguments, we should output some error message if no argument is passed.

Also please update the help command with arguments (maybe an example call for the command)

Reply:
Great suggestions! I made sure to add them :D it was a good call to update the help command as well